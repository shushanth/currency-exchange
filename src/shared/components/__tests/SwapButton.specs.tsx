import React from 'react';
import renderer from 'react-test-renderer';

import { CESwapButton } from '../index';

describe('<CESwapButton />', () => {
  test('tree snapshots match', () => {
    const swapButtonTree = renderer
      .create(
        <CESwapButton
          swapState={false}
          onSwapInvoke={swapState => swapState}
        />,
      )
      .toJSON();
    expect(swapButtonTree).toMatchSnapshot();
  });
});
