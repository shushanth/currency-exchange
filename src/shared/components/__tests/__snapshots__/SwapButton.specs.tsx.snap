// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CESwapButton /> tree snapshots match 1`] = `
<button
  className="sc-fzozJi mvuNo"
>
  <div>
    ↓
  </div>
  <div>
    ↑
  </div>
</button>
`;
